Lib/bookmarks.py

```diff
@@ -148,3 +148,37 @@ class FolderBookmark(StoredBookmark):
     """
 
     __type__ = "public.folder"
+
+
+def AllBookmarks():
```

Comment:
```suggestion
def all_bookmarks():
```
In Python, classes are CamelCase while functions and methods are snake_case.

Reply:
Thanks for the update. I wasn't aware of that convention. Ironically, in an earlier working version, I called the function all_bookmarks.